pages/nft-book-store/status/[classId].vue

```diff
@@ -687,6 +687,22 @@ function getOrdersTableActionItems (purchaseListItem: any) {
     }])
   }
 
+  if (purchaseListItem.shippingStatus === 'pending') {
+    actionItems.push([{
+      label: 'Handle Shipping',
+      icon: 'i-heroicons-paper-airplane',
```

Comment:
<img width="48" alt="image" src="https://github.com/likecoin/nft-book-press/assets/17264716/566bfe41-90eb-4cba-9b70-608371956318">

```suggestion
      icon: 'i-heroicons-truck',
```